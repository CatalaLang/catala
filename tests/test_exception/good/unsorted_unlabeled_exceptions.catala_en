## Test

```catala
declaration scope A:
  context x content integer
  context y content integer

scope A:
  exception
  definition x equals 1

  exception
  definition y equals 2

  definition x equals 0

  definition y equals 3
```
