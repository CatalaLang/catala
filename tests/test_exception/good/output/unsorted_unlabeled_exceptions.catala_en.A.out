[RESULT] Computation successful! Results:
[RESULT] x = 1
[RESULT] y = 2
